pom.xml

```diff
@@ -193,6 +195,59 @@
                     <compatibleSinceVersion>2.0</compatibleSinceVersion>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>1.12.1</version>
```

Comment:
This configuration is already provided in the parent POM if you check in a blank `.mvn_exec_node` file:

https://github.com/jenkinsci/plugin-pom/blob/a60e3494e8447aefed562deace5d222e4777413a/pom.xml#L1128-L1198